### PR TITLE
Display commit hash and fix pre-existing lint error

### PR DESCRIPTION
**Pull request by thom-at-redhat**
# Description

Display commit hash in logfile.  Additionally, fix a pre-existing lint error in events_test.go

Fixes # (issue)

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by ldjebran — COMMENTED** [review]
looks good , left one comment

File: main.go [hunk 1]
```diff
@@ -155,6 +157,21 @@ func main() {
 	signal.Notify(interruptSignal, os.Interrupt, syscall.SIGTERM)
 
 	initDependencies()
+
+	if buildInfo, ok := debug.ReadBuildInfo(); ok {
```
[the remark this answers is not in the document]
Comment (thom-at-redhat) [reply main.go]: @ldjebran added, please double check and resolve conversation if acceptable

**Review by ldjebran — APPROVED** [review]
LGTM

File: main.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (thom-at-redhat) [reply main.go]: Djebran added LGTM, so resolving question

**Review by de1987 — APPROVED** [review]

**Pull request merged**